package/MDAnalysis/core/groups.py

```diff
@@ -1541,3 +1586,24 @@ def residues(self):
         residuesclass = self.level.child.plural
         return residuesclass(self._u._topology.resindices[self],
                              self._u)
+
+# Define relationships between these classes
+# with Level objects
+Atomlevel = levels.Level('atom', Atom, AtomGroup)
```

Comment:
I think module level attributes are meant to be ALL CAPS BECAUSE THEY'RE VERY LOUD, (or for stylistic reasons)